src/webgpu/util/compare.ts

```diff
@@ -381,12 +395,20 @@ type SerializedComparatorSkipUndefined = {
   data?: SerializedExpectation;
 };
 
-// Serialized forms of 'value', 'packed', and 'bitcast' are intentionally
-// omitted, so should not be put into the cache. Attempting to will cause a
-// runtime assert.
+/** SerializedComparatorAlwaysPass is the serialized type of `alwaysPass` comparator. */
+type SerializedComparatorAlwaysPass = {
+  kind: 'alwaysPass';
+  data: string;
```

Comment:
There's no need for this to be called `data`, right? `reason` would be a better name for this field.

Reply:
Done